RNTester/js/RNTesterApp.ios.js

```diff
@@ -236,6 +378,13 @@ const styles = StyleSheet.create({
   exampleContainer: {
     flex: 1,
   },
+  bottomNavbar: {
+    bottom: 0,
+    width: '100%',
```

Comment:
Concepts of '%' doesn't exist in react native, you can do something like `width= Dimensions.get('window').width;` , or just `width:'stretch'` whichever works better.

Reply:
I think react native does? https://stackoverflow.com/questions/33939974/make-view-80-width-of-parent-in-react-native

Reply:
Oh shit! I wasn't aware of it. I feel my life has been a lie 😱

Reply:
Resolving this then.